src/modules/utils/motordrivercontrol/drivers/DRV8711/drv8711.cpp

```diff
@@ -15,7 +15,7 @@
 #define REGWRITE    0x00
 #define REGREAD     0x80
 
-DRV8711DRV::DRV8711DRV(std::function<int(uint8_t *b, int cnt, uint8_t *r)> spi, char d) : spi(spi), designator(d)
+DRV8711DRV::DRV8711DRV(std::function<void(uint8_t *b, int cnt, uint8_t *r)> spi, char d) : spi(spi), designator(d)
```

Comment:
Why was the signature of this changed? it seems unnecessary and forces changes to many files, including some outstanding PRs that probably rely on existing signatures.